corehq/apps/reports/util.py

```diff
@@ -84,7 +84,7 @@ def _create_temp_user(user_id):
             return temp_user
         return None
 
-    user_ids = user_ids if user_ids and user_ids[0] else None
+    user_ids = user_ids if bool(user_ids) else None
```

Comment:
Curious though, can user_ids be populated with an empty string. I thought it would be either a blank list or list with values?